packages/preset-mini/src/_rules/transition.ts

```diff
@@ -51,9 +49,9 @@ export const transitions: Rule<Theme>[] = [
 
       else if (d != null) {
         return {
-          'transition-duration': theme.duration?.[d] ?? h.time(d),
           'transition-property': theme.transitionProperty?.DEFAULT,
           'transition-timing-function': theme.easing?.DEFAULT,
+          'transition-duration': theme.duration?.[d] ?? h.time(d),
```

Comment:
I think it is best to keep the original location because it will affect the inconsistency of the output file and the problem of cache not being hit.